tests/development/test_base_component.py

```diff
@@ -687,8 +691,8 @@ def test_docstring(self):
 
     def test_events(self):
         self.assertEqual(
-            self.ComponentClass().available_events,
-            ['restyle', 'relayout', 'click']
+            hasattr(self.ComponentClass(), 'available_events'),
+            False
```

Comment:
Maybe this test should be renamed to make clearer what it's testing now

Reply:
> Maybe this test should be renamed

good call -> https://github.com/plotly/dash/pull/550/commits/ec9aaeee4bd109febbfd8f86e626f36f1ae6af58